itests/kit/init.go

```diff
@@ -40,6 +41,12 @@ func init() {
 
 	build.InsecurePoStValidation = true
 
+	// NOTE: If we want the payment channel itests to pass that use a
+	// block time of 5*millisecond, we need to set the consistent broadcast
+	// delay to something lower than that block time.
+	// todo: configure such a low delay only for paych tests.
+	build.CBDeliveryDelay = 2 * time.Millisecond
```

Comment:
Can we fully disable this for most tests? wdpost tests for example use 2ms block time, and it would be good to not slow them down even more.

Reply:
I completely disabled consistent broadcast for itests. That being said, it would be great if we can at least enable it on a smoke consensus or deals itests with a decent block time (if there is any) just as a sanity-check that this doesn't break anything in the future.